pkg/trait/knative_service_test.go

```diff
@@ -19,8 +19,10 @@ package trait
 
 import (
 	"path/filepath"
+	"reflect"
 	"testing"
 
+	routev1 "github.com/openshift/api/route/v1"
```

Comment:
Is it correct to use the openshift route object to test the knative service annotation ?

Reply:
No I should have used serving instead